ext/mysql2/extconf.rb

```diff
@@ -22,6 +22,10 @@ def add_ssl_defines(header)
   $CFLAGS << ' -DNO_SSL_MODE_SUPPORT' if has_no_support
 end
 
+# MariaDB 10.2+ Needs mysql/mysql_version.h loaded to get the constants we need
+have_header('mysql/mysql_version.h')
+
+
```

Comment:
Looks good - nit: please move to line 114, and use the `prefix` variable from the path test at line 100.